### PR TITLE
Update logging architecture

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/roadmap/issues/3518

This PR updates the logging architecture page to reflect the current state of the logging platform.

### CODE REVIEW

File: src/content/vintage/getting-started/observability/logging/architecture/index.md [hunk 1]
```diff
@@ -35,10 +35,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scrapping agents run on the management cluster and your workload clusters alike. There is different tools for different purposes:
+    - Promtail is used in older versions to retrieve the container and kubernetes audit logs.
+    - Alloy is used in newer versions to retrieve the container and kubernetes audit logs.
     - Grafana Agent is used to retrieve the kubernetes events.
```
Comment (pipo02mix) [inline src/content/vintage/getting-started/observability/logging/architecture/index.md:42]: But according to the picture, Alloy replaces Promtail and Grafana in CAPI v29? or only promtail.

Comment (TheoBrigitte) [reply src/content/vintage/getting-started/observability/logging/architecture/index.md:42]: It only replaces Promtail for now, but will eventually replace Grafana agent in the future.

File: src/content/vintage/getting-started/observability/logging/architecture/index.md [hunk 2]
```diff
@@ -35,10 +35,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scraping agents run on the management cluster and your workload clusters. There are different tools for different purposes:
+    - Promtail is used in older versions to retrieve the container and kubernetes audit logs.
+    - Alloy is used in newer versions to retrieve the container and kubernetes audit logs.
```
Comment (TheoBrigitte) [inline src/content/vintage/getting-started/observability/logging/architecture/index.md:40]:
```suggestion
    - Promtail is used in older Giant Swarm releases to retrieve the container and kubernetes audit logs.
    - Alloy is used in newer Giant Swarm releases to retrieve the container and kubernetes audit logs.
```

File: src/content/vintage/getting-started/observability/logging/architecture/index.md [hunk 3]
```diff
@@ -35,10 +35,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scrapping agents run on the management cluster and your workload clusters alike. There is different tools for different purposes:
+    - Promtail is used in older versions to retrieve the container and kubernetes audit logs.
```
[the remark this answers is not in the document]
Comment (TheoBrigitte) [reply src/content/vintage/getting-started/observability/logging/architecture/index.md]: Good point, I updated that.

**Comment by pipo02mix** [comment]
@TheoBrigitte can we point to the new docs? We are about to switch. This one will perfectly fit as a child of -> [here](https://docs.giantswarm.io/overview/observability/)

**Comment by TheoBrigitte** [comment]
@pipo02mix Where is the new doc ?

**Comment by pipo02mix** [comment]
My suggestion -> https://docs.giantswarm.io/overview/observability/logging

Example of something similar-> https://docs.giantswarm.io/overview/fleet-management/cluster-management/

File: src/content/overview/observability/logging/architecture/index.md [hunk 4]
```diff
@@ -35,10 +38,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scraping agents run on the management cluster and your workload clusters. There are different tools for different purposes:
+    - Promtail is used in older Giant Swarm releases to retrieve the container and kubernetes audit logs.
```
Comment (pipo02mix) [inline src/content/overview/observability/logging/architecture/index.md:42]:
```suggestion
    - Promtail is used in older Giant Swarm releases to retrieve the container and `Kubernetes` audit logs.
```

File: src/content/overview/observability/logging/architecture/index.md [hunk 5]
```diff
@@ -35,10 +38,21 @@ In this diagram, you can see that we run the following tools in each management
...
+    - Alloy is used in newer Giant Swarm releases to retrieve the container and kubernetes audit logs.
```
Comment (pipo02mix) [inline src/content/overview/observability/logging/architecture/index.md:43]:
```suggestion
    - Alloy is used in newer Giant Swarm releases to retrieve the container and `Kubernetes` audit logs.
```

File: src/content/overview/observability/logging/architecture/index.md [hunk 6]
```diff
@@ -35,10 +38,21 @@ In this diagram, you can see that we run the following tools in each management
...
     - Grafana Agent is used to retrieve the kubernetes events.
 
+### Release compatibility
+
+Release|Alloy|Promtail|Grafana Agent|
```
Comment (pipo02mix) [inline src/content/overview/observability/logging/architecture/index.md:48]: You need to add Promtail in the `.vale/config/accept.txt` words

**Review by pipo02mix — APPROVED** [review]
nice!

File: src/content/overview/observability/logging/architecture/index.md (hunk 4 above)
Comment (stone-z) [reply src/content/overview/observability/logging/architecture/index.md:42]: IMO the code formatting is not correct in this context (same for other comment)

Comment (TheoBrigitte) [reply src/content/overview/observability/logging/architecture/index.md:42]:
```suggestion
    - Promtail is used in older Giant Swarm releases to retrieve the container and Kubernetes audit logs.
```

File: src/content/overview/observability/logging/architecture/index.md (hunk 6 above)
Comment (TheoBrigitte) [reply src/content/overview/observability/logging/architecture/index.md:48]: added

**Review by QuentinBisson — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Great work

**Review by AndiDog — APPROVED** [review]

**Pull request merged**